### PR TITLE
User log in 

### PR DESCRIPTION
**Pull request by SophieSjoberg**
## What we have learned
We had a bit of trouble with setting up the environment but now we finally are able to continue and Cucumber is up and running

### CODE REVIEW

File: features/step_definitions/user_log_in.rb [hunk 1]
```diff
@@ -0,0 +1,38 @@
+Given("The following user exist") do |table|
+  table.hashes.each do |user|
+    FactoryBot.create(:user, user)
+  end
+end
+
+Given("I visit sign in page") do
+  # visit '/users/sign_in'
```
Comment (diraulo) [inline features/step_definitions/user_log_in.rb]: Remove dead code when you're done :)

File: features/user_log_in.feature [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+  Scenario: User is notified that he provided invalid password
+    Given I visit sign in page
+    And I fill in the "Email" with "sophie.sjoberg@gmail.com"
+    And I fill in the "Password" with my "wrongpassword"
+    And I click "Log in" button
+    Then I should be on the sign in page
+    And I should see "Invalid Email or password."
+
+    Scenario: User is notified that he provided invalid email
```
Comment (diraulo) [inline features/user_log_in.feature]: This Scenario block should be aligned with the previous one... pay attention to code formatting 😉

**Review by diraulo — DISMISSED** [review]
Minor stuff to fix and we can ship this! 😉

File: features/step_definitions/user_log_in.rb (hunk 1 above)
Comment (SophieSjoberg) [reply features/step_definitions/user_log_in.rb]: 👍

File: features/user_log_in.feature (hunk 2 above)
Comment (SophieSjoberg) [reply features/user_log_in.feature]: 👍

**Pull request merged**